wpilibj/src/main/java/edu/wpi/first/wpilibj/trajectory/TrajectoryParameterizer.java

```diff
@@ -256,6 +257,7 @@ public static Trajectory timeParameterizeTrajectory(
     return new Trajectory(states);
   }
 
+  @SuppressWarnings("PMD.AvoidThrowingRawExceptionTypes")
```

Comment:
PMD is right though... if we create a subclass of RuntimeException then users can differentiate between a path generation exception and something worse.

Reply:
Created a subclass of RuntimeException in the latest commit.